tools/wizards/mnecpp/test/wizard.json

```diff
@@ -0,0 +1,141 @@
+{
+    "version": 1,
+    "supportedProjectTypes": [ "Qt4ProjectManager.Qt4Project" ],
+    "id": "R.MNE-CPP Test",
+    "category": "I.MNE-CPP",
+    "trDescription": "Creates a plain MNE-CPP Test.",
+    "trDisplayName": "MNE-CPP Test",
+    "trDisplayCategory": "MNE-CPP",
+    "icon": "../../global/consoleapplication.png",
+    "enabled": "%{JS: [ %{Plugins} ].indexOf('QmakeProjectManager') >= 0 }",
+
+    "options":
+    [
+        { "key": "ProjectFile", "value": "%{JS: '%{ProFile}'}" },
+        { "key": "ProFile", "value": "%{JS: Util.fileName('%{ProjectDirectory}/%{ProjectName}', 'pro')}" },
+        { "key": "CppFileName", "value": "%{JS: '%{ProjectName}.' + Util.preferredSuffix('text/x-c++src')}" }
+    ],
+
+    "pages":
+    [
+        {
+            "trDisplayName": "Test Location",
+            "trShortTitle": "Location",
+            "typeId": "Project"
+        },
+        {
+            "trDisplayName": "Test Details",
+            "trShortTitle": "Details",
+            "typeId": "Fields",
+            "data" :
+            [
+				{
+                    "name": "author",
+                    "trDisplayName": "Your full name:",
+                    "mandatory": true,
+                    "type": "LineEdit",
+                    "data": {}
+                }, 
+
+				{
+                    "name": "eMail",
+                    "trDisplayName": "Your E-Mail address:",
+                    "mandatory": true,
+                    "type": "LineEdit",
+                    "data": {}
+                },   
+
+                {
+                    "name": "Month",
+                    "trDisplayName": "Current Month:",
+                    "mandatory": true,
+                    "type": "LineEdit",
+                    "data": {}
+                },   
+                
+                {
+                    "name": "Year",
+                    "trDisplayName": "Current Year:",
+                    "mandatory": true,
+                    "type": "LineEdit",
+                    "data": {}
+                },
+                
+                {
+                    "name": "testDescription",
+                    "trDisplayName": "Brief description:",
+                    "mandatory": true,
+                    "type": "LineEdit",
+                    "data": {}
+                },	
+
+                {
+                    "name": "TestClassName",
+                    "trDisplayName": "Test Class Name:",
+                    "mandatory": true,
+                    "type": "LineEdit",
+                    "data": {}
```

Comment:
Can we add default values for the class name such as `Test<YourTestClassName>` and for the project name `test_<YourTestName>`

Reply:
I could change it for the class name, but it seems that changing the file name is not straight forward.